includes/wp-admin/class-health-check.php

```diff
@@ -263,27 +263,55 @@ public static function is_webfinger_endpoint_accessible() {
 	 * @return array The filtered information
 	 */
 	public static function debug_information( $info ) {
+		$info['activitypub'] = array(
+			'label'  => \__( 'ActivityPub', 'activitypub' ),
+			'fields' => array(),
+		);
+
 		$actor = Actors::get_by_id( \get_current_user_id() );
 
-		$info['activitypub'] = array(
-			'label'  => __( 'ActivityPub', 'activitypub' ),
-			'fields' => array(
-				'webfinger'  => array(
-					'label'   => __( 'WebFinger Resource', 'activitypub' ),
-					'value'   => Webfinger::get_user_resource( wp_get_current_user()->ID ),
-					'private' => false,
-				),
-				'author_url' => array(
-					'label'   => __( 'Author URL', 'activitypub' ),
-					'value'   => $actor->get_url(),
-					'private' => false,
-				),
-				'author_id'  => array(
-					'label'   => __( 'Author ID', 'activitypub' ),
-					'value'   => $actor->get_id(),
-					'private' => false,
-				),
-			),
+		if ( $actor && ! is_wp_error( $actor ) ) {
```

Comment:
Maybe we could still show these rows, but with an empty value or a message that we couldn't get that information because the current user doesn't have a Fediverse profile?

Reply:
Never mind, I think this is good as-is.